experiments/calibration/make_training_locations.jl

```diff
@@ -5,30 +5,44 @@ using ClimaComms
 using ClimaCore
 ClimaComms.@import_required_backends
 
+
 """
-    make_training_locations(nelements)
+    diagnostics_lat_lon(nelements)
```

Comment:
Note: I re-added this function because we need it in observationseries_era5.jl 

We could return lats and longs from make_training_locations, but that would be confusing I think, because training_locations contains coordinates after the mask is applied, and lats longs before